archlinuxcn/trojan-go-git/lilac.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+
+from lilaclib import *
+
+def post_build():
+    git_add_files('PKGBUILD')
+    git_commit()
+    update_aur_repo() 
```

Comment:
Don't update AUR repo since it's a VCS package.

Reply:
I maintain that package on AUR and have granted `lilac` as comaintainer.

Reply:
Per AUR Guidelines, you should not update PKGBUILD if it's only a VCS change.

Reply:
Done. `lilac.py` removed. 
I picked up this bad habit from `qv2ray-dev-git` package. Later I will file another PR to fix that.